File: examples/openapi-sdk-petstore/kotlin/sdk/com/example/petstore/models/Order.kt
package com.example.petstore.models

import kotlinx.datetime.Instant
import kotlinx.serialization.Serializable

@Serializable
public data class Order(
    public val id: Long? = null,
    public val petId: Long? = null,
    public val quantity: Int? = null,
    public val shipDate: Instant? = null,
    public val status: Order_Status? = null,
    public val complete: Boolean? = null,
)

File: examples/openapi-sdk-petstore/swift/sdk/Models/Order.swift
import Foundation

public struct Order: Codable {
    public let id: Int64?
    public let petId: Int64?
    public let quantity: Int32?
    public let shipDate: Date?
    public let status: Order_Status?
    public let complete: Bool?

    public init(
        id: Int64? = nil,
        petId: Int64? = nil,
        quantity: Int32? = nil,
        shipDate: Date? = nil,
        status: Order_Status? = nil,
        complete: Bool? = nil
    ) {
        self.id = id
        self.petId = petId
        self.quantity = quantity
        self.shipDate = shipDate
        self.status = status
        self.complete = complete
    }
}
